diff -up brltty-6.0/Bindings/Java/Makefile.in.0001~ brltty-6.0/Bindings/Java/Makefile.in
--- brltty-6.0/Bindings/Java/Makefile.in.0001~	2019-07-13 17:12:57.150714074 +0200
+++ brltty-6.0/Bindings/Java/Makefile.in	2019-07-13 17:13:43.464711827 +0200
@@ -58,7 +58,7 @@ $(JAVA_JNI_FILE): bindings.$O $(API_NAME
 	$(MKLIB:<name>=$(API_NAME)) $@ bindings.$O $(API_LDFLAGS)
 
 bindings.$O:
-	$(CC) $(LIBCFLAGS) $(JAVA_JNI_FLAGS) -o $@ -c $(SRC_DIR)/bindings.c
+	$(CC) $(LIBCFLAGS) -I$(BLD_TOP)$(PGM_DIR) $(JAVA_JNI_FLAGS) -o $@ -c $(SRC_DIR)/bindings.c
 
 JAVA_JAR_FILE = $(API_NAME).jar
 jar: $(JAVA_JAR_FILE)
